test-cypress/integration/ttyg/agent-list.spec.js

```diff
@@ -24,6 +24,7 @@ describe('TTYG agent list', () => {
 
     it('Should be able to toggle agents panel', () => {
         TTYGStubs.stubChatsListGet();
+        TTYGStubs.stubChatGet();
         TTYGStubs.stubAgentListGet();
         // Given I have opened the ttyg page
         TTYGViewSteps.visit();
```

Comment:
Add wait for GET chat response after each visit

Reply:
done